gulp-tasks/test.js

```diff
@@ -242,6 +242,26 @@ function findBadMDExtensions() {
 }
 
 /**
+ * Chec to ensure there an o JavaSCript files in the content/ folder.
+ *
+ * Note: the promise will always be fulfilled, even if it fails.
+ *
+ * @return {promise} An empty promsie
+ */
+ function findJavaSCriptFiles() {
+  return new Promise(function(resolve, reject) {
+    var contentPath = GLOBAL.WF.src.content;
+    var patterns = ['**/*.js', '!**/_code/*.js', '!**/event-map.js'];
+    var files = wfHelper.getFileList(contentPath, patterns);
+    files.forEach(function(filename) {
+      summary.files++;
+      logError(filename.filePath, '\n\tJS files must be in a directory named /_code.')
```

Comment:
I'd prefer to drop it for consistency right now as we're not using it anywhere else. 

As for the `_code/` directory, all JS files checked into the Google Source Repo need to be reviewed by someone with JS readability, which makes that CL harder to get approved. I'd rather discourage people from doing it unless it's a special case.